modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-creating-and-using-boot-sources.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling a volume snapshot as a boot source
+
+You can import a volume snapshot as a boot source by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
```

Comment:
Following the comment about the title I also think this can change to `You can enable volume snapshot boot sources`